src/html/transfers.html

```diff
@@ -619,12 +617,13 @@ <h3 class="transfer__amount">${window.utils.formatLargeNum(
       if (!(window.ethInitialized && window.nearInitialized)) return;
 
       const transfers = await window.transfers.get();
-      const params = Object.keys(window.urlParams.get());
+      const currentParams = Object.keys(window.urlParams.get());
+      const allParams = ['new', 'erc20', 'erc20n']
```

Comment:
"If current params includes all params, hide recent transfers. Otherwise, show recent transfers."

These names seem really confusing. `allParams` especially. Maybe `irrelevantParams` instead? "If params are irrelevant, hide. Else, show."

Reply:
`allParams` are the pages of the app : `new, erc20 and erc20n`, so we want to display transfers if not one of those pages is active. Maybe `appUrlPaths` instead then ?